cli/dinghy/check_env.rb

```diff
@@ -28,9 +28,21 @@ def expected
       "DOCKER_MACHINE_NAME" => machine.name,
     }
   end
+  
+  def shell
+    begin
+        return `echo $SHELL`.strip
+    rescue
+        return
```

Comment:
this branch should return the empty string `""`, or something, so that it doesn't cause an error when calling `.end_with?` below